go/cmd/vtctldclient/command/vreplication/mount/mount.go

```diff
@@ -0,0 +1,175 @@
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Mount %s registered successfully\n", req.Name)
+	return nil
+}
+
+var unregister = &cobra.Command{
+	Use:                   "unregister",
+	Short:                 "Unregister a mount",
+	Example:               `vtctldclient --server localhost:15999 Mount Unregister ext1`,
+	DisableFlagsInUseLine: true,
+	Aliases:               []string{"Unregister"},
+	Args:                  cobra.ExactArgs(1),
+	RunE:                  commandUnregister,
+}
+
+func commandUnregister(cmd *cobra.Command, args []string) error {
+	cli.FinishedParsing(cmd)
+
+	req := &vtctldatapb.MountUnregisterRequest{
+		Name: args[0],
+	}
+	_, err := common.GetClient().MountUnregister(common.GetCommandCtx(), req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Mount %s unregistered successfully\n", req.Name)
+	return nil
+}
+
+var show = &cobra.Command{
+	Use:                   "show",
+	Short:                 "Show attributes of a mount",
+	Example:               `vtctldclient --server localhost:15999 Mount Show ext1`,
+	DisableFlagsInUseLine: true,
+	Aliases:               []string{"Show"},
+	Args:                  cobra.ExactArgs(1),
+	RunE:                  commandShow,
+}
+
+func commandShow(cmd *cobra.Command, args []string) error {
+	cli.FinishedParsing(cmd)
+
+	req := &vtctldatapb.MountShowRequest{
+		Name: args[0],
+	}
+	resp, err := common.GetClient().MountShow(common.GetCommandCtx(), req)
+	if err != nil {
+		return err
+	}
+	data, err := json.Marshal(resp)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%s\n", string(data))
+	return nil
+}
+
+var list = &cobra.Command{
+	Use:                   "list",
+	Short:                 "List all mounts",
+	Example:               `vtctldclient --server localhost:15999 Mount List`,
+	DisableFlagsInUseLine: true,
+	Aliases:               []string{"List"},
+	Args:                  cobra.NoArgs,
+	RunE:                  commandList,
+}
+
+func commandList(cmd *cobra.Command, args []string) error {
+	cli.FinishedParsing(cmd)
+
+	req := &vtctldatapb.MountListRequest{}
+	resp, err := common.GetClient().MountList(common.GetCommandCtx(), req)
+	if err != nil {
+		return err
+	}
+	if err != nil {
+		return err
+	}
+	data, err := json.Marshal(resp)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%s\n", string(data))
+	return nil
+}
+
+func registerCommands(root *cobra.Command) {
+	root.AddCommand(mount)
+	addRegisterFlags(register)
+	mount.AddCommand(register)
+	mount.AddCommand(unregister)
+	mount.AddCommand(show)
+	mount.AddCommand(list)
+}
+
+func addRegisterFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&mountOptions.TopoType, "topo-type", "", "Topo server implementation to use")
+	cmd.Flags().StringVar(&mountOptions.TopoServer, "topo-server", "", "Topo server address")
+	cmd.Flags().StringVar(&mountOptions.TopoRoot, "topo-root", "", "Topo server root path")
+	for _, flag := range []string{"topo-type", "topo-server", "topo-root"} {
+		cmd.MarkFlagRequired(flag)
+	}
```

Comment:
It's still 3 lines, so IMO it's easier to read if we mark each one as required immediately following its declaration.